chapters/annotations.tex

```diff
@@ -72,6 +72,19 @@ \subsection{Class Description and Revision History}\label{annotation-info-revisi
 <a href="modelica:/MultiBody.Joints.Revolute#info">Revolute</a>
 \end{lstlisting}
 
+\subsubsection{Style sheet}
+
+A top-level package may contain an annotation \lstinline!styleSheet="modelica:/Package/Resources/style.css"!, which provides a cascading stylesheet.
+The given stylesheet is used when displaying the \lstinline!info! and \lstinline!revisions! annotations found in the \lstinline!Documentation! annotations.
+The provided stylesheet shall not use type or universal selectors.
+\begin{nonnormative}
+It is recommended to use class and id selectors with a \lstinline!$NameOfLibrary$-! prefix to avoid collisions when the content is included in a larger context.
```

Comment:
Markup (need a tiny listings language definition as well):
```suggestion
It is recommended to use \lstinline![language=css]!class! and \lstinline![language=css]!id! selectors with a \lstinline[language=css]!$NameOfLibrary$-! prefix to avoid collisions when the content is included in a larger context.
```